README.md

```diff
@@ -365,9 +383,16 @@ How to choose between all these features? Here are some considerations:
         #[serde(with = "clickhouse::serde::time::datetime")]
         dt: OffsetDateTime,
     }
+    // or if you prefer chrono: 
+    #[derive(Row, Serialize, Deserialize)]
+    struct MyRow {
+        ts: u32,
```

Comment:
`ts` and `dt` are shown in one `struct MyRow,` so let's show `chrono` also in that one row; it seems redundant to copy-paste the `ts` variant here.

Same for other examples